libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -798,9 +817,9 @@ static CK_RV prvRsaKeyAttParse( const CK_ATTRIBUTE * pxAttribute,
         CK_BBOOL xBool = ( CK_BBOOL ) CK_FALSE;
         int32_t lMbedTLSResult = 0;
         CK_RV xResult = CKR_OK;
-        mbedtls_ecp_keypair * pxKeyPair = ( mbedtls_ecp_keypair * ) pxMbedContext->pk_ctx;
 
-        configASSERT( pxKeyPair != NULL );
+        configASSERT( ( mbedtls_ecp_keypair * ) pxMbedContext->pk_ctx != NULL );
```

Comment:
Same comment about the assert expression change. I think the previous version is more readable